docs/Glossary/index.md

```diff
@@ -0,0 +1,136 @@
+# Glossary
+
+This glossary provides definitions of commonly used terms and phrases, organized by application or function.
+
+---
+
+## Media Management Applications
+
+**Radarr**, **Sonarr**, and **Lidarr** are applications designed to manage your media libraries:
+
+- **Radarr**: For movies
+- **Sonarr**: For TV shows
+- **Lidarr**: For music
+
+### Custom Format
+
+A feature that matches specific aspects of a release based on information extracted from its name. Users choose the matching criteria, and Custom Formats can be assigned scores on a per-[Quality Profile](#quality-profile) basis.
```

Comment:
```suggestion
A feature that uses pattern matching on a release based on the information implied through the release name on the corresponding indexer during a search. Users can customize patterns to match a Custom Format, and Custom Formats can then be assigned scores individually in each [Quality Profile](#quality-profile). After the release has been imported, the Custom Format score will be updated to match the properties of the actual media file, rather than strictly by its original naming alone.
```
Sheesh that was a lot :P

Reply:
Added a comma.